test/Crate/Test/DBAL/Functional/Schema/SchemaManagerTest.php

```diff
@@ -231,4 +242,19 @@ protected function assertHasTable($tables, $tableName)
         $this->assertTrue($foundTable, "Could not find new table");
     }
 
+    public function testListTableIndexes()
```

Comment:
Do we also need to adapt the testListTableColumns test?

Reply:
why that?

Reply:
Because of the changes within https://github.com/crate/crate-dbal/pull/77/files#diff-3bf31e6e2bf51e359758d534f3e71950R53-R80

Afaik this should have some effect on the `listTableColumns()` function, shouldn't it?

Reply:
The relevant changes are reflected by https://github.com/crate/crate-dbal/pull/77/files#diff-4b98eaee1c9a7dd15dca8757838b017fR125-R134. Not sure what else you're thinking of.